library/network/src/lib/y2firewall/firewalld/api/zones.rb

```diff
@@ -131,8 +131,7 @@ def add_interface(zone, interface, permanent: permanent?)
         #   modifies the permanent configuration
         # @return [Boolean] True if the interface was removed from the zone
         def remove_interface(zone, interface, permanent: permanent?)
-          modify_command("--zone=#{zone}", "--remove-interface=#{interface}",
-            permanent: permanent)
+          modify_command("--zone=#{zone}", "--remove-interface=#{interface}", permanent: permanent)
```

Comment:
What's changed here? I only see the `permanent` argument being moved to the same line, right?

Reply:
Right ;)